theseus/geometry/se3.py

```diff
@@ -285,14 +285,26 @@ def exp_map(
 
             jac[:, 3:, 3:] = jac[:, :3, :3]
 
+            minus_one_by_twelve = torch.full(
+                near_zero.shape,
+                -1 / 12.0,
+                dtype=sine_by_theta.dtype,
+                device=sine_by_theta.device,
+            )
```

Comment:
I just tested.
<pre><code>minus_one_by_twelve = torch.tensor(-1 / 12.0, 
                                   dtype=sine_by_theta.dtype, 
                                   device=since_by_theta.device)</code></pre>
should work for `torch.where()`.